nupic/research/frameworks/pytorch/test_utils/fake_data.py

```diff
@@ -19,8 +19,8 @@
 # ----------------------------------------------------------------------
 
 from torch.utils.data import DataLoader
+from torchvision import transforms
```

Comment:
Not really changing anything in this file, maybe revert to original

Reply:
need to add blank line at the end of the original file, because it gives a style error otherwise.

see here: https://github.com/numenta/nupic.research/blob/master/nupic/research/frameworks/pytorch/test_utils/fake_data.py